--- a/CMakeLists.txt	2026-01-17 16:23:42.056121262 -0500
+++ b/CMakeLists.txt	2026-05-19 14:17:07.001803545 +0200
@@ -13,7 +13,7 @@
 # cmake --build . --config Release
 # cmake --install .
 
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.20)
 project(texmaker VERSION 6.0.1 LANGUAGES CXX C)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_BINARY_DIR})
 
@@ -27,7 +27,7 @@
     if (${COMPILEUSB} STREQUAL "yes")
         set(INSTALL_DIR ${CMAKE_SOURCE_DIR}/deploy/linux_portable)
     else()
-        set(INSTALL_DIR "/usr")
+        set(INSTALL_DIR "/app")
     endif()
 elseif (WIN32)
     if (${COMPILEUSB} STREQUAL "yes")
@@ -387,6 +387,7 @@
 set(CMAKE_AUTORCC ON)
 
 find_package(Qt6 REQUIRED COMPONENTS Gui Concurrent Core Core5Compat Network PrintSupport Qml Widgets Xml LinguistTools)
+find_package(Qt6CorePrivate REQUIRED)
 
 qt_standard_project_setup()
 
