--- a/CMakeLists.txt	2026-01-17 16:23:42.056121262 -0500
+++ b/CMakeLists.txt	2026-01-17 16:26:39.468127580 -0500
@@ -13,7 +13,7 @@
 # cmake --build . --config Release
 # cmake --install .
 
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.20)
 project(texmaker VERSION 6.0.1 LANGUAGES CXX C)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_BINARY_DIR})
 
@@ -387,6 +387,7 @@
 set(CMAKE_AUTORCC ON)
 
 find_package(Qt6 REQUIRED COMPONENTS Gui Concurrent Core Core5Compat Network PrintSupport Qml Widgets Xml LinguistTools)
+find_package(Qt6CorePrivate REQUIRED)
 
 qt_standard_project_setup()
 
